components/branchs-dropdown.tsx

```diff
@@ -60,11 +72,21 @@ export default function BranchsDropdown({
       <ReactSelect
         key={`select_repo-${repoId}`}
         isDisabled={disabled || !options?.length}
-        options={options}
+        options={options?.sort(sortOptions)}
         value={option}
         onChange={onChangeSelect}
         placeholder={t("forms.select-placeholder")}
       />
+      {!option && (
+        <div className="mt-2">
+          <p className="text-gray">
+            {t("support-text-branch")}{" "}
+            <a href="https://discord.gg/layerx" target="_blank">
```

Comment:
I think we have this value on a variable somewhere, if its not: add it to a constant please

Reply:
I also created and adjusted constant from other links